<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="0" failures="0" skipped="1" tests="6" time="6.850" timestamp="2025-09-03T07:50:55.277114" hostname="pkrvm7jw40e0xgp"><testcase classname="vpn_api.test_admin_flow" name="test_admin_flow" time="1.785" /><testcase classname="vpn_api.test_flow" name="test_basic_flow" time="0.831" /><testcase classname="vpn_api.test_payments" name="test_payments_crud_flow" time="0.859" /><testcase classname="vpn_api.test_peers" name="test_peers_crud_flow" time="0.872" /><testcase classname="vpn_api.test_unit" name="test_register_and_login" time="0.816" /><testcase classname="vpn_api.test_unit" name="test_tariff_crud" time="0.273"><skipped type="pytest.skip" message="Admin user not available">/home/runner/work/vpn-api/vpn-api/vpn_api/test_unit.py:28: Admin user not available</skipped></testcase></testsuite></testsuites>
